aerospike-stubs/aerospike.pyi

```diff
@@ -309,6 +309,19 @@ TXN_STATE_VERIFIED: Literal[1]
 TXN_STATE_COMMITTED: Literal[2]
 TXN_STATE_ABORTED: Literal[3]
 
+CDT_SELECT_MATCHING_TREE: Literal[0]
+CDT_SELECT_VALUES: Literal[1]
+CDT_SELECT_MAP_KEY_VALUES: Literal[1]
+CDT_SELECT_MAP_KEYS: Literal[2]
+CDT_SELECT_NO_FAIL: Literal[16]
```

Comment:
In C, no fail is defined:

> 	AS_CDT_SELECT_NO_FAIL = 0x10

16 is right, but I would define it at 0x10 for consistency.